app-cdr/daa2iso/daa2iso-0.1.7e-r1.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+DESCRIPTION="Convert CD images from daa/gbi to iso"
+HOMEPAGE="http://aluigi.altervista.org/mytoolz.htm"
+SRC_URI="http://aluigi.altervista.org/mytoolz/${PN}.zip -> ${P}.zip"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux ~x64-macos"
+
+DEPEND="app-arch/unzip"
+
+S="${WORKDIR}"
+
+src_compile() {
+	emake -C src -f "${FILESDIR}/Makefile"
```

Comment:
Nice to know! Yes, seems that emake doesn't imply correct `CC` ([portage/bin/ebuild-helpers/emake](https://github.com/gentoo/portage/blob/a514908c7fd29e668dd775d7a32c949dbc8fe7d7/bin/ebuild-helpers/emake)) so `tc-getCC()` should be used explicitly. 

There is a ton of places in the tree where it isn't set, though, and in some places it's set to ${CC} for some reason ([games-action/xpilot#L42](https://github.com/gentoo/gentoo/blob/ce8e64d919c82c84edac51007e37c0580d8e5c8f/games-action/xpilot/xpilot-4.5.5.ebuild#L42)).